job-service-scheduled-executor-container/pom.xml

```diff
@@ -171,6 +171,11 @@
                                     </inline>
                                 </assembly>
                             </build>
+                            <run>
+                                <env>
+                                    <CAF_WORKER_INPUT_QUEUE>jobservicescheduler-in</CAF_WORKER_INPUT_QUEUE>
+                                </env>
+                            </run>
```

Comment:
This doesn't make any sense.  Did you mean to put the `env` element inside `build`, for it to become the default?

Reply:
Moved into build